tests/unit/image_manager/test_image_inspector.py

```
@@ -0,0 +1,55 @@
+"""Unit tests for image inspector."""
+
+import pytest
+
+from ansible_navigator.command_runner import Command
```

Flake8 (7.3.0)

[error] 'ansible_navigator.command_runner.Command' imported but unused

(F401)